### DISCUSSION TITLE
The `border-r-inherit` class does not exist.

### DISCUSSION THREAD

**Discussion by jan-dh** [tailwindlabs/tailwindcss#7473] [Help]
Copy pasted it straight from the docs, I'm on Tailwind 3.0.22. I keep getting this error:

```The `border-r-inherit` class does not exist. If `border-r-inherit` is a custom class, make sure it is defined within a `@layer` directive.```

Link to the docs: https://tailwindcss.com/docs/border-color

<img width="528" alt="image" src="https://user-images.githubusercontent.com/5801781/154044745-568f282f-8aba-4c6b-83d9-745ee0c91229.png">

**Comment by Bredhis** [comment]
Hello Jan,

I have had the same problem with `border-inherit`. I've added custom colours, like so:
https://tailwindcss.com/docs/customizing-colors#using-custom-colors

So after encountering the problem I've added `inherit: 'inherit',`, like they did for _transparent_ and _currentColor_.

Maybe that's exactly what you're missing too.

**Comment by ZachSaucier** [comment]
This seems to be true of all values like `border-inherit` and `border-current` also. My work around was to use a custom value, e.g. `border-[currentColor]`
